LICENSE.md

```diff
@@ -1,12 +1,12 @@
 Dragonfly Business Source License 1.1
 
-<u>License</u>: BSL 1.1
+<u>License</u>: BUSL 1.1
 
 <u>Licensor</u>: DragonflyDB, Ltd.
 
 <u>Licensed Work</u>: Dragonfly including the software components, or any portion of them, and any modification.
 
-<u>Change Date</u>: March 15, 2028
+<u>Change Date</u>: August 15, 2028
```

Comment:
It's confusing but the change date does not have to be 4 years, it can any date.
This text sets the global condition:
```
Effective on the Change Date, or the forth anniversary of the first
publicly available distribution of a specific version of the Licensed
Work under this License, whichever comes first
```

Reply:
whichever comes first. So I use the Change date to make sure we have at least 4 years for every version we release. 
v1.16 has less than 4 years, because this PR has not been comitted